docs/source/maintenance/permissions.rst

```diff
@@ -247,6 +256,38 @@ can_set_my_account_detail
     :linenos:
     :lines: 10-44
 
+can_compare_and_set_detail
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Allows safe setting `account <../core_concepts/glossary.html#account>`__ detail.
+
+The `permission <../core_concepts/glossary.html#permission>`__ allows safe setting details to other accounts. Another way to safe set detail without can_compare_and_set_detail permission is to grant `can_compare_and_set_my_account_detail`_ permission to someone. In order to grant, `transaction <../core_concepts/glossary.html#transaction>`__ creator should have `can_grant_can_compare_and_set_my_account_detail`_ permission.
+
+.. Note:: Transaction creator can always safe set detail for own account even without that permission.
```

Comment:
How does this comply with the following?
https://github.com/hyperledger/iroha/blame/6649a54f95fc86c104112db7c97f87d02a728db5/docs/source/api/commands.rst#L836

Reply:
Not sure I understood correctly

Reply:
Fixed